src/core/adapters/notion.ts

```diff
@@ -30,7 +28,9 @@ type NotionTicketResponse = {
  * Turns a slugged page ID without dashes into a dasherized RFC 4122 UUID.
  * UUID-Format: 96ec637d-e4b0-4a5e-acf3-8d4d9a1b2e4b
  */
-function uuid(slugId: string) {
+function uuid(slugId: string | null) {
+  if (!slugId) return null;
```

Comment:
Hm, I to be honest don't get the point of this change 🤔. Maybe I'm missing something... But why would we want to spread checks for the ID being `null` over multiple functions? How about just an early return inside the `scan` function before even calling `uuid`?